Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
ah, that is what you meant. ok, but that is already the case with the code above it seems as it is separate link tag(?)